File: problem-5/main.lua
function IsEvenlyDivisible(n)
    for i=1,20 do
        if n % i ~= 0 then
            return false
        end
    end
    return true
end
i = 1
while true do
    if IsEvenlyDivisible(i) then
        break
    end
    i = i + 1
end
print(i)

File: problem-5/main.cs
using System;

namespace ProjectEuler {
    class ProblemFive {
        static void Main(string[] args){
            int i = 1;
            while (true) {
                if (IsEvenlyDivisible(i)) {
                    break;
                }
                i++;
            }
            Console.WriteLine(i);
        }
        static bool IsEvenlyDivisible(int n){
            for (int i = 1; i < 21; i++) {
                if (n % i != 0) {
                    return false;
                }
            }
            return true;
        }
    }
}